akka-persistence/src/main/scala/akka/persistence/PersistentView.scala

```diff
@@ -361,6 +361,10 @@ trait PersistentView extends Actor with Snapshotter with Stash with StashFactory
     override def recoveryRunning: Boolean = false
 
     override def stateReceive(receive: Receive, message: Any): Unit = message match {
+      case ReplayedMessage(p) ⇒
+        // we can get ReplayedMessage here if it was stashed by user during replay
+        // unwrap the payload
+        PersistentView.super.aroundReceive(receive, p.payload)
```

Comment:
user asked for it, see #17896, and it is obviously not good to emit the internal `ReplayedMessage` wrapper